Moving the template dependency to the ShopBundle (#13656)
Sylius/Sylius: UPGRADE-2.0.md

* `use_webpack` option was removed from the `sylius_ui` configuration, and the Webpack has become the only module
  bundler provided by Sylius.
* `use_webpack` twig global variable was removed. Webpack is always used now, and there is no need to check for it.

## Theming

> If you are running your shop in the headless mode the `sylius/theme-bundle` is no longer installed as it has been moved to the shop bundle.

Reviewer: ```suggestion
Dependency on `sylius/theme-bundle` is moved from CoreBundle to ShopBundle and it will no longer be installed if you're running your shop in headless mode.
```

> * The `themeName` form field has been removed from the Channel, if the shop bundle is not installed.

Reviewer: Rephrasing is needed, my take (which is not ideal as well)

```suggestion
* Channel's `themeName` form field existence is made optional and depends `ShopBundle` presence.
```
